src/hooks/tracking.ts

```diff
@@ -1069,3 +1096,71 @@ export const useTracking = (trackingOptions = { separator: '|' }) => {
     },
   }
 }
+
+export const useMemberShipCardDetails = (memberId: string | undefined = '') => {
+  if (memberId == null) memberId = ''
```

Comment:
avoid using `==`

could modify at line 1123, `memberId: memberId ?? ''`

Reply:
Oh! Dealing with undefined values twice

The expression 'memberId == null' is equivalent to 'memberId === null | memberId === undefined'.
Okey,I'll avoid it in the future.

This is a good suggestion for use  "memberId: memberId ?? ''